terjira.gemspec

```diff
@@ -20,13 +20,14 @@ Gem::Specification.new do |spec|
   spec.executables   = ["bin/jira"].map { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "thor", "~> 1.0.1"
+  spec.add_dependency "thor", "~> 1.0"
   spec.add_dependency "jira-ruby", "~> 2.1"
   spec.add_dependency "activesupport", ">= 4"
 
-  spec.add_dependency "tty-table", "~> 0.11.0"
-  spec.add_dependency "tty-prompt", "~> 0.20.0"
-  spec.add_dependency "tty-spinner", "~> 0.9.3"
+  spec.add_dependency "tty-table", ">= 0.12"
+  spec.add_dependency "tty-prompt", ">= 0.23"
+  spec.add_dependency "tty-spinner", ">= 0.9"
+  spec.add_dependency "strings", ">= 0.2"
```

Comment:
Keeps these dependency a little more loose so it allows for easier upgrade next time around. [Last merged PR was me](https://github.com/keepcosmos/terjira/pull/101), a year ago, for the same reason ;)